cmd/sops/main.go

```diff
@@ -356,8 +368,6 @@ func main() {
 	app.Action = func(c *cli.Context) error {
 		if c.Bool("verbose") {
```

Comment:
I'm not too familiar with your cli library, but why does this happen twice / in two different places?

Reply:
SOPS has several subcommands (`sops`, `sops keyservice`, `sops updatekeys`, etc) and in code a subcommand is represented as a function that takes a `*cli.Context` that contains all CLI flags. This check happens in two different places because it has to be made for all subcommands. It could be put into a setupLogging function that's called from all subcommands and passed the context.